### PR TITLE
Feeds documentation: Add benkow panels tracker

### PR DESCRIPTION
**Pull request by certxlm**
Hello,

Here is an updated version of the YAML feed configuration file that integrates one of the Benkow feeds.
Source page: http://benkow.cc/passwords.php

Feel free to give me feedback if my commit needs to be changed.

Thanks

### CODE REVIEW

File: intelmq/etc/feeds.yaml [hunk 1]
```diff
@@ -1898,3 +1898,30 @@ providers:
...
+            filter_text: "id;type;url;ip;date;"
+            filter_type: blacklist
+            delimiter: ";"
+            skip_header: True
+            columns:
+              - __IGNORE__
+              - malware.name|__IGNORE__
+              - destination.url
+              - destination.fqdn|destination.ip|__IGNORE__
+              - time.source|__IGNORE__
```
Comment (wagner-certat) [inline intelmq/etc/feeds.yaml:1927]: The malware name can't be invalid, so we don't need to ignore anything. For the IP column we can set the required flag to false. For the time, I don't see a reason to ignore it. (Tested locally and it worked for me)
```suggestion
            columns:
              - __IGNORE__
              - malware.name
              - destination.url
              - destination.fqdn|destination.ip
              - time.source
              columns_required:
              - false
              - true
              - true
              - false
              - true
```

**Review by wagner-certat — CHANGES_REQUESTED** [review]
Thanks for the addition! I have some minor remarks/comments on some details.

File: intelmq/etc/feeds.yaml [hunk 2]
```diff
@@ -1898,3 +1898,30 @@ providers:
...
+        collector:
+          module: intelmq.bots.collectors.http.collector_http
+          parameters:
+            http_url: http://benkow.cc/export.php
+            name: __FEED__
+            provider: benkow.cc
+        parser:
+          module: intelmq.bots.parsers.generic.parser_csv
+          parameters:
+            type: malware
```
[the remark this answers is not in the document]
Comment (certxlm) [reply intelmq/etc/feeds.yaml:1917]: Where can I find the full taxonomy please?
It's not easy to fit this into the given list as it's about malware/botnet panels.
The closest is probably `c2-server`. What do you think about it?

[the remark this answers is not in the document]
Comment (wagner-certat) [reply intelmq/etc/feeds.yaml:1917]: The upstream taxonomy is as https://github.com/enisaeu/Reference-Security-Incident-Taxonomy-Task-Force/blob/master/working_copy/humanv1.md and https://github.com/enisaeu/Reference-Security-Incident-Taxonomy-Task-Force/blob/master/working_copy/machinev1 for JSON-format. And in the intelmq documentation here: https://intelmq.readthedocs.io/en/latest/dev/data-format.html#classification (please note that this is the latest-version, not maintenance, of the docs)

c2-server looks fine for me. The users still can adapt it if they choose other options.

[the remark this answers is not in the document]
Comment (certxlm) [reply intelmq/etc/feeds.yaml]: Thanks !
Fixed !

**Review by wagner-certat — APPROVED** [review]

File: intelmq/etc/feeds.yaml [hunk 3]
```diff
@@ -1898,3 +1898,34 @@ providers:
...
+            type: c2-server
+            delimiter: ";"
+            skip_header: True
+            columns:
+              - __IGNORE__
+              - malware.name
+              - destination.url
+              - destination.fqdn|destination.ip
+              - time.source
+              columns_required:
```
Comment (wagner-certat) [inline intelmq/etc/feeds.yaml:1926]:
```suggestion
            columns_required:
```

File: intelmq/etc/feeds.yaml [hunk 4]
```diff
@@ -1898,3 +1898,34 @@ providers:
             http_password: <your password>
         parser:
           module: intelmq.bots.parsers.fireeye.parser
+  Benkow:
+    Malware Panels Tracker:
+      description: Benkow Panels tracker is a list of fresh panel from various malware. The feed is available on the webpage: http://benkow.cc/passwords.php
```
Comment (wagner-certat) [inline intelmq/etc/feeds.yaml:1903]:
```suggestion
      description: "Benkow Panels tracker is a list of fresh panel from various malware. The feed is available on the webpage: http://benkow.cc/passwords.php"
```
YAML doesn't like the colon

**Review by wagner-certat — APPROVED** [review]

**Comment by wagner-certat** [comment]
Thanks for the addition!

If you know of any other feeds which could be added to the list, your contributions are welcome! Also private feeds, as long as the information about their existence can be shared.

There's a bug in the documentation-generating code, which I'll fix directly. I'm also squashing the commits into one for merging.

**Comment by wagner-certat** [comment]
Fixed the mentioned (unrelated) bug in 9058c9cc2347e93067430cc44e6ef2feadcbb46d

**Pull request merged**